ngraph/frontend/paddlepaddle/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "paddlepaddle_ngraph_frontend")
+
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cc)
+file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp ${CMAKE_CURRENT_SOURCE_DIR}/src/*.h)
+file(GLOB_RECURSE LIBRARY_PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
+
+if (TARGET ext_protobuf)
+#    add_dependencies(ngraph::paddlepaddle_ngraph_frontend ext_protobuf)
+    add_dependencies(${TARGET_NAME} ext_protobuf)
+endif()
+
+set(${TARGET_NAME}_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
do we need it?

Reply:
The original idea is to have PaddlePaddle frontend as a shared library, which can be potentially linked to user's applications. So that user app can cast 'InputModel' to 'PDPDInputModel' and call some paddlepaddle-specific functions
We'll check if this is still the case. If not, then I guess we can really try to implement this as 'module', not shared lib.
If yes, then public headers, archive, "shared library" settings should be kept as is

Reply:
Discussed, decided to keep it as is (shared library, see other comments)